src/interface.jl

```diff
@@ -51,7 +51,7 @@ function Base.getindex{T<:Integer}(b::Buffer, r::UnitRange{T})
 end
 
 Base.setindex!(b::Buffer, lines::Array, r::CappedRange) =
-    set_line_slice(b, r.start, r.stop, true, true, lines)
+    set_line_slice(b, r.start, r.stop, true, true, convert(Array{ASCIIString, 1}, lines))
```

Comment:
This is kind of a hack since the latest version of Julia makes Array{Any, 1} the default instead of Array{None, 1}.